website/src/examples/imagery-layer.js

```diff
@@ -0,0 +1,141 @@
+import React, {Component} from 'react';
+import {GITHUB_TREE} from '../constants/defaults';
+import App from 'website-examples/wms/app';
+import styled from 'styled-components';
+
+import {makeExample} from '../components';
+
+const DemoContainer = styled.div`
+.selected-feature-info {
+  position: absolute;
+  z-index: 10;
+  font-size: 12px;
+  line-height: 1.25;
+  background: #fff;
+  box-shadow: 2px 2px 4px rgb(0 0 0 / 20%);
+  padding: 8px;
+  max-width: 320px;
+  white-space: pre-wrap;
+}
+`;
+
+const SERVICES = {
+  'Terrestris (OpenStreetMap)': {
+    serviceUrl: `https://ows.terrestris.de/osm/service`,
+    serviceType: 'wms',
+    defaultLayers: ['OSM-WMS'],
+    viewState: {
```

Comment:
I suppose this could be auto calculated based on layer extents in getCapabilities - something to look into during metadata normalization in loaders.gl.

Reply:
I have it hard coded here for the same reason as the `defaultLayers` - metadata sometimes take too long to load.

But I agree it would be nice to have it as part of the metadata schema.